infrastructure/pcluster-manager.yaml

```diff
@@ -200,6 +196,21 @@ Resources:
       PayloadFormatVersion: 2.0
       TimeoutInMillis: 30000
 
+  PCMApiGatewayAccessLog:
+    Type: AWS::Logs::LogGroup
+    Properties:
+      RetentionInDays: 90
```

Comment:
suggestion: we could specify a log group name so that it's not totally random and autogenerated by CloudFormation

Reply:
I'm ok with it, from CloudFormation documentation it was not clear that the outcome would be like that!
Thanks